packages/onboarding/src/components/RecommendedRibbon/RecommendedRibbon.js

```diff
@@ -3,10 +3,10 @@
  */
 import { __ } from '@wordpress/i18n';
 
-const localPartners = [ 'mercadopago' ];
+const localPartners = [ 'woo-mercado-pago-custom' ];
```

Comment:
I'm thinking that since this is a component available for external consumption, we should extra some this `localParterns` logic to the consumer and instead use an `isLocal` prop.  Thoughts?

Reply:
I definitely agree with this. I'm not entirely clear on what _"local"_ means in this case. By consumer are you referring to the WCCOM config, or the gateway @joshuatf ?

Reply:
Good question.  Something like `isLocal` in the gateway API might make sense, but I'm also unclear on what "local" means in this case or how we'd use it going forward with other gateways.

@octaedro @pmcpinto  do you have any more information about this?

Reply:
This ribbon is potentially shown before the gateway is installed, so I believe it would have to go in the WCCOM config. If it's truly a one-time thing that's not going to change, then putting it in the consuming component would work as well. Although it feels a bit less clean.

Reply:
Agreed.  Will wait on @octaedro or @pmcpinto for some context.

Reply:
@joshuatf @joelclimbsthings in some countries we need to display a Recommended ribbon to highlight local partners. Eg: https://d.pr/i/pihuM1

I think that currently we're only doing this in Brazil, but probably we'll expand this approach to more countries in the upcoming months. Lmk if you have more questions.

Reply:
Thanks, @pmcpinto!

>I think that currently we're only doing this in Brazil, but probably we'll expand this approach to more countries in the upcoming months.

I think this tells us what we need to know.  If we're introducing more, we should probably make this part of the data source / API.

Reply:
This begs the question for me whether we should be moving all of the recommended logic into that config, as we have `RECOMMENDED_GATEWAY_IDS` in the consuming component right now as well. @joshuatf 

Also, @pmcpinto , the link you provided shows the text "Recommended" on the banner for Mercado Pago, but our current implementation shows "Local Partner."

![image](https://user-images.githubusercontent.com/444632/123833773-8eba7380-d8bb-11eb-86bb-f71ce3f86846.png)

Just wanted to confirm that we want to show the altered text for Mercado Pago (Local Partner), which we may expand to others going forward.

Reply:
>This begs the question for me whether we should be moving all of the recommended logic into that config

I think that would be a good idea.  Not quite sure on the specifics, but one thing that popped in my mind was a priority (integer) of recommendation so that we don't need to worry about the order of items in the array.

Reply:
> Also, @pmcpinto , the link you provided shows the text "Recommended" on the banner for Mercado Pago, but our current implementation shows "Local Partner."

@joelclimbsthings thanks for spotting this. The copy should be "Local Partner". I don't know why is showing "Recommended"